integration-cli/docker_cli_stats_test.go

```diff
@@ -39,5 +39,20 @@ func (s *DockerSuite) TestCliStatsNoStream(c *check.C) {
 		statsCmd.Process.Kill()
 		c.Fatalf("stats did not return immediately when not streaming")
 	}
+}
+
+func (s *DockerSuite) TestStatsContainerNotFound(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
I saw @jhowardmsft added it to all the other similar tests so I just copy pasted